src/M6Web/Bundle/ElasticsearchBundle/DependencyInjection/M6WebElasticsearchExtension.php

```diff
@@ -22,86 +22,135 @@ class M6WebElasticsearchExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        $config        = $this->processConfiguration($configuration, $configs);
```

Comment:
I think we should not align assigment 'cause if you add a new variable with name lenth higher than existing ones, you have to reindent whole block and made some noise on diff